sss/tests/test_report.py

```diff
@@ -61,12 +61,23 @@ def test_report_to_db(setup_and_teardown_package):
     report_df.update_person.replace({np.nan: None}, inplace=True)
     report_df["update_date"] = report_df["upload_status"].str.split(" ").str[1]
     print(report_df["update_date"])
```

Comment:
Looks like this was left from before, but we should probably remove this debugging print statement.